src/main/java/org/fogbowcloud/saps/engine/utils/ExecutionScriptTagUtil.java

```diff
@@ -15,6 +15,7 @@
 	
 	protected static final String ERROR_MSG__TYPE_NOT_FOUND = "Execution Script Tag type not found.";
 	protected static String DEFAULT_SCRIPT_TAG_JSON_PATH = "resources/execution_script_tags.json";
+	protected static String DEFAULT_ENDTOEND_SCRIPT_TAG_JSON_PATH = "resources/execution_endtoend_script_tags.json";
```

Comment:
this is something we definitely don't want to add to our code. I'm fine to accept this PR if you add a feature to have a CLI or API to submit scripts